### PR TITLE
fix naming in document notification

### PR DESCRIPTION
**Pull request by petrus7**

### CODE REVIEW

File: geonode/locale/zh/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1077,8 +1077,8 @@ msgid "A Document was commented on"
 msgstr "已评论一个文件"
 
 #: geonode/documents/__init__.py:33
-msgid "Document for Map"
-msgstr "地图文件"
+msgid "Rating for Document"
+msgstr "文件评级"
```
Comment (giohappy) [inline geonode/locale/zh/LC_MESSAGES/django.po:1081]: @petrus7 Google translator? :)

Comment (petrus7) [reply geonode/locale/zh/LC_MESSAGES/django.po:1081]: Yes

Comment (giohappy) [reply geonode/locale/zh/LC_MESSAGES/django.po:1081]: I hope we're not writing something wrong!

**Comment by petrus7** [comment]
close in favour of 
https://github.com/GeoNode/geonode/pull/6085

**Pull request closed without merging**